cmd/iam_apikey_create.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyCreateItemOutput egoscale.APIKey
+
+func (o *apiKeyCreateItemOutput) toJSON()  { outputJSON(o) }
+func (o *apiKeyCreateItemOutput) toText()  { outputText(o) }
+func (o *apiKeyCreateItemOutput) toTable() { outputTable(o) }
+
+// apiKeyCreateCmd represents the create command
+var apiKeyCreateCmd = &cobra.Command{
+	Use:     "create <APIKey description>",
```

Comment:
I think the description is more relevant in a flag like `-d` and `--description`

Reply:
I agree with you, we will probably rename "Description" for "Name" in the API